samples/app/src/main/java/com/pingidentity/samples/app/davinci/DaVinciViewModel.kt

```diff
@@ -9,13 +9,10 @@ package com.pingidentity.samples.app.davinci
 
```

Comment:
Typo in the copyright header - missing space in `PingIdentity`...

Reply:
Updated